src/backend/joanie/tests/core/test_api_admin_orders.py

```diff
@@ -190,15 +198,46 @@ def test_api_admin_orders_retrieve(self):
                 },
                 "main_invoice": {
                     "balance": float(order.main_invoice.balance),
+                    "children": [
+                        {
+                            "balance": float(credit_note.balance),
+                            "children": [],
+                            "created_on": credit_note.created_on.isoformat().replace(
+                                "+00:00", "Z"
+                            ),
+                            "invoiced_balance": float(credit_note.invoiced_balance),
+                            "recipient_address": (
+                                f"{credit_note.recipient_address.full_name}\n"
+                                f"{credit_note.recipient_address.full_address}"
+                            ),
+                            "reference": credit_note.reference,
+                            "state": credit_note.state,
+                            "transactions_balance": float(
+                                credit_note.transactions_balance
+                            ),
+                            "total": float(credit_note.total),
+                            "total_currency": settings.DEFAULT_CURRENCY,
+                            "type": credit_note.type,
+                            "updated_on": credit_note.updated_on.isoformat().replace(
+                                "+00:00", "Z"
+                            ),
+                        }
+                    ],
                     "created_on": order.main_invoice.created_on.isoformat().replace(
                         "+00:00", "Z"
                     ),
-                    "state": order.main_invoice.state,
+                    "invoiced_balance": float(order.main_invoice.invoiced_balance),
```

Comment:
```suggestion
                    "invoiced_balance": D(order.main_invoice.invoiced_balance),
```
I think we should be cautious while using `float` as it might change the value ("rounding" the value), instead of parsing a `Decimal`. What is your opinion ?

Reply:
The serializer must return a float value here. With `D`, you'll get a `Decimal` class instance.